packages/app/background-jobs-common/src/periodic-jobs/AbstractPeriodicJob.ts

```diff
@@ -49,7 +50,9 @@ export abstract class AbstractPeriodicJob {
       singleConsumerMode: {
         enabled: false,
         exclusiveLockSuffix: options.singleConsumerMode?.exclusiveLockSuffix ?? 'EXCLUSIVE',
-        lockTimeout: options.intervalInMs * 2,
+        lockTimeout: options.schedule.intervalInMs
+          ? options.schedule.intervalInMs * 2
+          : DEFAULT_LOCK_TIMEOUT,
```

Comment:
If we use cron option, `lockTimeout` will always be the default value, is it fine?

Reply:
not super sure about it, but what would be a better alternative? you can still specify an explicit value if needed, not sure if we need a separate value for specifying a default

Reply:
Mmm, not sure, either, we can go with it for now and if we see it can be a problem then we can add a new value for it